### PR TITLE
[plugin.image.dumpert] 1.0.4+matrix.1

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
Made a python 3 version based upon my python 2 version.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.image.dumpert/addon.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <extension point="xbmc.addon.metadata">
+	<summary lang="en_GB">Watch funny pictures from Dumpert.nl (dutch)</summary>
+	<description lang="en_GB">Watch funny pictures from Dumpert.nl (dutch)</description>
+	<disclaimer lang="en_GB">For bugs, requests or general questions visit the Dumpert.nl thread on the Kodi forum.</disclaimer>
+	<summary lang="nl_NL">Bekijk grappige plaatjes van Dumpert.nl (dutch)</summary>
+	<description lang="nl_NL">Bekijk grappige plaatjes van Dumpert.nl (dutch)</description>
+	<disclaimer lang="nl_NL">Bugs of andere feedback op deze plugin kan geplaatst worden in de Dumpert.nl thread op het Kodi forum.</disclaimer>
+    <language>nl</language>
+    <platform>all</platform>
+    <license>GPL-2.0-or-later</license>
```
Comment (basrieter) [inline plugin.image.dumpert/addon.xml]: The license type does not match the on in license.txt  and the plugin of the `krypton` branch?

**Review by basrieter — CHANGES_REQUESTED** [review]
Please have a look at the license of this version. It seems to be different? If you update it, please use the PDX code (https://spdx.org/licenses/).

File: plugin.image.dumpert/addon.xml (hunk 1 above)
Comment (skipmodea1) [reply plugin.image.dumpert/addon.xml]: Yeah that's correct, i didn't do pull requests for the licence change. It is changed in my git for my addons however, so next time there is a real change, the licence change will be in the PR.

Comment (basrieter) [reply plugin.image.dumpert/addon.xml]: The I would suggest to stick with the original license in the addon.xml and make the complete change in the next PR. Now they are inconsistent, within Matrix, but also between Matrix and other Kodi versions.

Comment (skipmodea1) [reply plugin.image.dumpert/addon.xml]: Do i need to change something?

**Comment by micahg** [comment]
@basrieter not sure what the problem is (sorry). That page shows "GNU General Public License v2.0 or later" having a code of GPL-2.0-or-later, which is what I stuck in the addon.xml.

**Comment by basrieter** [comment]
The issue is that the Kodi Matrix version of the Dumpert add-on has a different license than the Dumpert add-on from per-Matrix. They should be the same, as it is the same add-on, just targeted at a new Kodi version. The LICENSE.txt still shows GPL3, just like the pre-Matrix version. So the addon.xml should als have GPL3 targeted: either GPL-3.0-only or GPL-3.0-or-later. 

So the addon.xml needs to be updated to reflect the correct license.txt.

File: plugin.image.dumpert/addon.xml (hunk 1 above)
Comment (basrieter) [reply plugin.image.dumpert/addon.xml]: The issue is that the Kodi Matrix version of the Dumpert add-on has a different license than the Dumpert add-on from per-Matrix. They should be the same, as it is the same add-on, just targeted at a new Kodi version. The LICENSE.txt still shows GPL3, just like the pre-Matrix version. So the addon.xml should als have GPL3 targeted: either GPL-3.0-only or GPL-3.0-or-later.

So the addon.xml needs to be updated to reflect the correct license info.

Comment (skipmodea1) [reply plugin.image.dumpert/addon.xml]: addon.xml is changed.

**Review by basrieter — APPROVED** [review]

**Comment by basrieter** [comment]
Thanks, merged it.

**Pull request merged**